app/services/subdomain.py

```diff
@@ -1,18 +1,23 @@
 from app import db, momblish
 from app.models import Subdomain
-from app.utils.errors import (
-    AccessDenied,
-    SubdomainTaken,
-    SubdomainInUse,
-    SubdomainLimitReached,
-)
+from app.utils.errors import SubdomainTaken, SubdomainInUse, SubdomainLimitReached
+
+subdomain_reserved_limits = {"free": 0, "paid": 5}
 
 
 class SubdomainCreationService:
     def __init__(self, current_user, subdomain_name=""):
         self.current_user = current_user
         self.subdomain_name = subdomain_name
 
+    def over_subdomain_limits(self):
```

Comment:
This is out of scope for this pr but made sense to bring this into line with tunnel limits